module/exporter-file/src/Domain/Command/CreateFileExportChannelCommand.php

```diff
@@ -30,18 +31,27 @@ class CreateFileExportChannelCommand implements CreateChannelCommandInterface
 
     protected string $exportType;
 
+    protected ?SegmentId $segmentId;
+
     /**
      * @param Language[] $languages
      */
-    public function __construct(ChannelId $id, string $name, string $format, string $exportType, array $languages = [])
-    {
+    public function __construct(
+        ChannelId $id,
+        string $name,
+        string $format,
+        string $exportType,
+        ?SegmentId $segmentId,
```

Comment:
Adding nullable parameter would make the change backward compatible. So it goes for other places where possible